src/resources/MachineLearning/ModelInformation/ModelInformationInterfaces.ts

```diff
@@ -0,0 +1,4 @@
+export interface MLModelInfo {
+    id: string;
+    info?: {};

```

Comment:
Do we have a type for the info?

Reply:
It seems we don't have properties in this _info_ object
![image](https://user-images.githubusercontent.com/52677246/67956056-e3c75300-fbc9-11e9-9eaf-1f3d2c8f8c67.png)